server/src/user/user.entity.ts

```diff
@@ -27,4 +28,7 @@ export class User extends CustomBaseEntity {
 
   @OneToMany(() => Application, (application) => application.user)
   applications: Application[];
+
+  @OneToMany(() => Comment, (comment) => comment.user)
```

Comment:
Needn't have this mapping!

Reply:
Hey Karthikk, if we get rid of these mappings, I won't be able to do things like this, right?
<img width="128" alt="image" src="https://user-images.githubusercontent.com/102998728/205684018-ce41b0b8-2776-4c63-9a97-99a5c0099878.png">

In that case, should I add a column on Comment called userId or something and add it manually?
Something like comment.userId = userId;
Thanks in advance!

Reply:
We can point it to `user.id`. Refer https://orkhan.gitbook.io/typeorm/docs/many-to-one-one-to-many-relations

Reply:
Got it, thank you!